src/org/armedbear/lisp/disassemble.lisp

```diff
@@ -149,7 +165,9 @@ SYS:*DISASSEMBLERS*."
               (if (or (java:jinstance-of-p classloader "org.armedbear.lisp.MemoryClassLoader")
                       (java:jinstance-of-p classloader "org.armedbear.lisp.FaslClassLoader"))
                   (disassemble-bytes 
-                   (java:jcall "getFunctionClassBytes" classloader class))
+		   (funcall (java:jfield "org.armedbear.lisp.Function" "FUNCTION_CLASS_BYTES") function) 
```

Comment:
Why this  better than calling `getFunctionClassBytes` on the classloader?  Perhaps it is an additional strategy, rather than a strict replacement?

Reply:
getFunctionClassBytes didn't work sometimes and I couldn't figure out why, but this works. Should be tracked down. By someone else ;-)

Reply:
So, we should modify by adding the strategy to an `(or (ignore-errors [CASE A]) (ignore-errors …` clause as it will break even less often…

Reply:
That's fine with me.